File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// BranchCrossEventsExport SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class BranchCrossEventsExportMakeContext
{
    public static function call(array $ctxmap, ?BranchCrossEventsExportContext $basectx): BranchCrossEventsExportContext
    {
        return new BranchCrossEventsExportContext($ctxmap, $basectx);
    }
}

File: py/branchcrosseventsexport_sdk/utility/make_context.py
# BranchCrossEventsExport SDK utility: make_context

from branchcrosseventsexport_sdk.core.context import BranchCrossEventsExportContext


def make_context_util(ctxmap, basectx):
    return BranchCrossEventsExportContext(ctxmap, basectx)
